src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<blockquote>To do that<code>,</code> we will start ...</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Changed in dfe233f7